assets/src/application/ui/display/confirm/useConfirm.tsx

```diff
@@ -19,19 +22,11 @@ const Confirm: React.FC<ConfirmProps> = ({ buttonProps, onConfirm, title }) => {
 		<>
 			<EspressoButton {...buttonProps} onClick={onOpen} />
 			<AlertDialog
-				cancelButton={
-					<Button ref={cancelRef} onClick={onClose}>
-						{__('No')}
-					</Button>
-				}
+				cancelButton={<Button buttonText={__('No')} ref={cancelRef} onClick={onClose} />}
```

Comment:
taking a fresh look now at the `Button` API, the `buttonText` doesn't help very much, because it's clear that the text is pertaining to button not to :crocodile: or anything else
I know, this is how it was before, just noticed that this is a bit verbose, we can use just `text` or pass it as `children`, dunno

Reply:
ya the previous API was because the button was handling multiple things, so I agree that this could be changed now.
might be better to just use children which allows for anything whereas a "text" prop implies a `string` only, but I'm ok with either.